python/pyspark/ml/wrapper.py

```diff
@@ -64,7 +64,7 @@ def _make_java_param_pair(self, param, value):
         param = self._resolveParam(param)
         java_param = self._java_obj.getParam(param.name)
         java_value = _py2java(sc, value)
-        return java_param.w(java_value)
+        return java_param, java_value
```

Comment:
This should not be changed.  I don't think you need to change it for its uses below.